packages/intranet-header-workspace/projects/intranet-header/src/lib/swisspost-intranet-header.component.html

```diff
@@ -107,32 +115,48 @@ <h1 class="m-0 font-weight-bold">{{ siteTitle }}</h1>
       </div>
     </div>
     <nav
-      *ngIf="hasNavbar === true"
+      *ngIf="hasNavbar === true && !condenseHeader"
       class="row justify-content-center top-navigation m-0 d-md-flex"
+      aria-label="navigation"
```

Comment:
This causes screenreaders to announce "navigation navigation" as the `<nav>` already prompts a (localized) announcement of navigation. When adding this statically, there is also no chance to localize it.

```suggestion
```

Reply:
I see. I only added it afterwards because sonar failed when it wasn't there. I will remove it again.